### PR TITLE
create go microservice

### PR DESCRIPTION
**Pull request by Damon259**

### CODE REVIEW

File: lib/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-tensorflow
```
Comment (free-y) [inline lib/requirements.txt:1]: Can we undo the change on this file which is irrelated to the go microservice?

Comment (Damon259) [reply lib/requirements.txt:1]: I don't see any changes.

File: backends/automl-go/build.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```
Comment (free-y) [inline backends/automl-go/build.sh:1]: Let's support deploying it with docker or Minikube.

File: backends/automl-go/components/graceful.go [hunk 3]
```diff
@@ -0,0 +1,318 @@
+/*
```
Comment (free-y) [inline backends/automl-go/components/graceful.go:1]: Can we import the lib with the package manager instead of cloning the source code?

**Comment by free-y** [comment]
Two months no update, close it. we can reopen it if needed.
